docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.components.scss

```diff
@@ -566,6 +566,10 @@ $civictheme-directional-link-dark-disabled-color: civictheme-color('neutral', 5)
 // Card/Event card.
 //
 $civictheme-event-card-border-radius: $civictheme-border-radius !default;
+$civictheme-event-card-image-height-mobile: 210px !default;
```

Comment:
here and below - we need to provide both height and width for images per breakpoint as a contained dimension may not be the same (e.g., on mobile, height could be contained, but on desktop - width; this applies to publication card; we are using a consistent way to specify these dimensions).